src/board.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file board.h
+ * @brief Board state for Numbers Royale
+ * 2 players and 2 sets of numbers that players choose from.
+ * Players take turns selecting a number from their set.
+ * The player with the larger number wins the round.
+ * Once a number is selected, it cannot be used again.
+ * Score is kept for each player.
+ */
+
+#include <set>
+#include <string>
+
+// Globals
+constexpr int P1 = 1;
+constexpr int P2 = 2;
```

Comment:
should this be part of the class?

Reply:
If I were to add them to the class, would I make them `static const`?

Reply:
`static constexpr`

Reply:
You might not need this, it could be part of the player class or something external